tests/calibration/test_calibration_node.py

```diff
@@ -614,13 +620,23 @@ def test_create_notebook_datetime_path(
                 # assert os.path.join(methods_node.nb_path, methods_node.node_id) in test_value
                 assert f"_{test_path}" in test_value
             if dirty and not error:
-                path_and_node_id = os.path.join(methods_node.nb_folder, methods_node.node_id)
-                assert path_and_node_id in test_value
+                # Check relative paths will not appear in the output path
+                if methods_node.nb_folder == ".." or methods_node.nb_folder == ".":
```

Comment:
Maybe increase the number of fixture to test here? Pass some relative and absolute and with `.` or `..`